tools/tenscan/frontend/src/components/modules/rollups/columns.tsx

```diff
@@ -0,0 +1,130 @@
+"use client";
+
+import { ColumnDef } from "@tanstack/react-table";
+import { DataTableColumnHeader } from "../common/data-table/data-table-column-header";
+import TruncatedAddress from "../common/truncated-address";
+import { formatNumber, formatTimeAgo } from "@/src/lib/utils";
+import Link from "next/link";
+import { EyeOpenIcon } from "@radix-ui/react-icons";
+import { Rollup } from "@/src/types/interfaces/RollupInterfaces";
+
+export const columns: ColumnDef<Rollup>[] = [
+  {
+    accessorKey: "ID",
+    header: ({ column }) => (
+      <DataTableColumnHeader column={column} title="ID" />
+    ),
+    cell: ({ row }) => {
+      return (
+        <div className="flex space-x-2">
+          <span className="max-w-[500px] truncate">{row.getValue("ID")}</span>
+        </div>
+      );
+    },
+    enableSorting: false,
+    enableHiding: false,
+  },
+  {
+    accessorKey: "Hash",
+    header: ({ column }) => (
+      <DataTableColumnHeader column={column} title="Hash" />
+    ),
+    cell: ({ row }) => {
+      return (
+        <Link
+          href={{
+            pathname: "/rollup/[hash]/batches",
+            query: { hash: row.getValue("Hash") },
+          }}
+          className="text-primary"
+        >
+          <TruncatedAddress address={row.getValue("Hash")} />
+        </Link>
+      );
+    },
+    enableSorting: false,
+    enableHiding: false,
+  },
+  {
+    accessorKey: "Timestamp",
+    header: ({ column }) => (
+      <DataTableColumnHeader column={column} title="Timestamp" />
+    ),
+    cell: ({ row }) => {
+      return (
+        <div className="flex space-x-2">
+          <span className="max-w-[500px] truncate">
+            {formatTimeAgo(row.getValue("Timestamp"))}
+          </span>
+        </div>
+      );
+    },
+    enableSorting: false,
+    enableHiding: false,
+  },
+  {
+    accessorKey: "L1Hash",
+    header: ({ column }) => (
+      <DataTableColumnHeader column={column} title="L1 Hash" />
+    ),
+    cell: ({ row }) => {
+      return <TruncatedAddress address={row.getValue("L1Hash")} />;
+    },
+    enableSorting: false,
+    enableHiding: false,
+  },
+  {
+    accessorKey: "FirstSeq",
+    header: ({ column }) => (
+      <DataTableColumnHeader column={column} title="First Sequencer" />
```

Comment:
This should be "First batch sequence number" and the Last Sequencer should become "Last batch sequence number"